oneapp-server/types/ApplicationProgramInfo.js

```diff
@@ -0,0 +1,52 @@
+const { gql } = require('apollo-server-express');
+
+const typeDef = gql`
+  extend type Application {
+    programInfo: ApplicationProgramInfo
+  }
+
+  type ApplicationProgramInfo {
+    "Applying for SNAP"
+    IS_FS_SELECTED: Boolean
+    "Applying for TANF"
+    IS_TF_SELECTED: Boolean
+    "Applying for General Assistance"
+    IS_GA_SELECTED: Boolean
+    "Has active SNAP case"
+    HAVE_ACTIVE_CASE_CURRENTLY: Boolean
+    "Case number(s) of active SNAP applications"
+    CURRENT_CASE_NUMBERS: String
+    "Has received SNAP previously"
+    HAD_ACTIVE_CASE_PREVIOULSY: Boolean
+    "Case number(s) of previous SNAP applications"
+    PREVIOUS_CASE_NUMBERS: String
+    "Spoken language of applicant"
+    SPOKEN_LANGUAGE: String
+    "Applicant needs accommodation"
+    NEED_ACCOMODATION: Boolean
+    "Needs accommodation type translator"
```

Comment:
nit: shift these fields to ApplicationContact as they pertain to the applicant?

Reply:
100% agree, but aiming to maintain 1:1 with the database / stored procedures.